rootfs.go

```diff
@@ -14,23 +14,76 @@
 package main
 
 import "fmt"
+import "net/http"
 import "io"
 import "os"
+import "strings"
 
 func doRootfs(imageFile string) error {
```

Comment:
In general I would rather refactor this a bit and move file handling and network handling to separate functions. This will case the code to be more readable IMHO.